Dockerfile.server

```diff
@@ -5,6 +5,21 @@ FROM node:5.5
 
 USER root
 
+# Install Xvfb and its dependencies needed to run Electron (Nightmare tests)
+RUN apt-get update && \
+    apt-get install -y \
+    libgtk2.0-0 \
+    libgconf-2-4 \
+    libasound2 \
```

Comment:
Yeah I could try to minimize the list. I started with the dependencies listed in https://github.com/segmentio/nightmare/issues/224#issuecomment-141575361 and added them based on errors I saw when attempting to run Electron in the container. I didn't want to use the entire list found in https://github.com/segmentio/nightmare/issues/224#issuecomment-163470030 or [the Electron docs](https://github.com/atom/electron/blob/master/docs/development/build-instructions-linux.md),  but maybe we can get a more minimal list.